src/Blaster.WebApi/Features/Shared/httpclient.js

```diff
@@ -1,6 +1,32 @@
 import jq from "jquery";
 
 const baseUrl = `${window.basePath}`;
+const axios = require('axios').default.create();
+
+axios.interceptors.request.use(
+    req => {
+        const internalHttpClient = new HttpClient();
+        console.log(req.url + " has been hit");

```

Comment:
Do we use UglifyJsPlugin in webpack to remove console.log statements from production env?

Reply:
Nope, will be removed, merely just for visual feedback when demonstrating/trying new things.

Reply:
Maybe we should add the uglify plugin to sleep better at night? :D

Reply:
Sounds reasonable! :)